packages/core/src/scxml.ts

```diff
@@ -497,7 +485,18 @@ function scxmlToMachine(
   scxmlJson: XMLElement,
   options: ScxmlToMachineOptions
 ): MachineNode {
-  const machineElement = scxmlJson.elements!.find(
+  if (!scxmlJson.elements) {
+    // If there is no SCXML element, this should call 'done.invoke' immediately.
```

Comment:
I'm not entirely sure if this is correct. According to SCXML:
> A conformant SCXML document must have at least one <state>, <parallel> or <final> child

I've found out that this conditional block is only executed for [the test 530](https://gitlab.com/scion-scxml/test-framework/-/blob/b46a10a1c3a3b1ca5c5cb4bb44ddb5c785611f41/test/w3c-ecma/test530.txml.scxml). And for it we get `scxmlJson` equal to:
```json
{
  "type": "element",
  "name": "content",
  "attributes": {
    "expr": "Var1"
  }
}
```
This test actually assigns the SCXML document to context (! 😱 ) and later uses it in `<invoke>` - I'm pretty sure our SCXML converted doesn't support this yet so this test should get disabled for now and those lines here might not be needed (or rather they should be replaced with other logic)